linkerd/app/inbound/src/direct.rs

```diff
@@ -28,31 +28,44 @@ pub struct RefusedNoIdentity(());
 #[error("a named target must be provided on gateway connections")]
 struct RefusedNoTarget;
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub(crate) struct Local {
+    pub port: u16,
+    pub permit: policy::Permit,
```

Comment:
nit: AFAICT these fields aren't actually accessed directly anywhere outside of this file (since they're exposed via `Param` impls), so can't this just be
```suggestion
    port: u16,
    permit: policy::Permit,
```